### PR TITLE
Removed Aura Wrappers for Flows

### PR DESCRIPTION
**Pull request by adityanaag3**
Updated LWC to be directly embedded in Flows, and updated the flows to use LWC instead of Aura Wrappers

### CODE REVIEW

File: es-space-mgmt/main/default/flows/createReservation.flow-meta.xml [hunk 1]
```diff
@@ -78,6 +78,7 @@
             <label>isLead</label>
         </rules>
     </decisions>
+    <description>Updated to LWC</description>
```
Comment (slychika) [inline es-space-mgmt/main/default/flows/createReservation.flow-meta.xml]: Create Reservation Flow with LWC.

File: es-space-mgmt/main/default/flows/spaceDesigner.flow-meta.xml [hunk 2]
```diff
@@ -94,29 +100,36 @@
             <targetReference>updateReservation</targetReference>
         </connector>
         <fields>
-            <name>designFormCmp</name>
-            <extensionName>c:designFormCmp</extensionName>
+            <name>spaceDesignFormLWC</name>
```
Comment (slychika) [inline es-space-mgmt/main/default/flows/spaceDesigner.flow-meta.xml]: Remove LWC from name

File: es-space-mgmt/main/default/flows/spaceDesigner.flow-meta.xml [hunk 3]
```diff
@@ -50,13 +49,20 @@
             <label>GoToRecordHome</label>
         </rules>
     </decisions>
+    <description>Updated with LWC</description>
```
Comment (slychika) [inline es-space-mgmt/main/default/flows/spaceDesigner.flow-meta.xml]: Make this a description of the component, not the tech.

File: es-space-mgmt/main/default/flows/createReservation.flow-meta.xml [hunk 4]
```diff
@@ -244,17 +245,17 @@
             <targetReference>assignCustomerLookup</targetReference>
         </connector>
         <fields>
-            <name>CustomerDetailscmp</name>
-            <extensionName>c:customerDetails</extensionName>
+            <name>reservationHelperFormLWC</name>
```
Comment (slychika) [inline es-space-mgmt/main/default/flows/createReservation.flow-meta.xml]: remove LWC from name

**Review by slychika — CHANGES_REQUESTED** [review]

**Review by slychika — APPROVED** [review]
Looks great, thanks for making those changes.

**Review by muenzpraeger — APPROVED** [review]

**Pull request merged**